.env.example

```
@@ -96,9 +96,9 @@ RESUME_KEYWORDS=resume,cv,curriculum
 OPENAI_API_KEY=
 # For OpenRouter, set OPENAI_BASE_URL=https://openrouter.ai/api/v1
 OPENAI_BASE_URL=
-OPENAI_MODEL=gpt-4o-mini
+OPENAI_MODEL=5o-mini
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OPENAI_MODEL key should go before the RESUME_KEYWORDS key

(UnorderedKey)